### PR TITLE
ImageNet Pytorch Workload Processing Fix

### PR DESCRIPTION
**Pull request by pomonam**
The aim of this PR is to match the performance with the ImageNet Jax workloads. Most importantly, the input normalization was fixed; I suspect the ResNet models were less impacted by this as it uses BatchNorm, but we should expect better results with the pre-selected LR and weight decay. Additionally, I added the followings.
- Fix RandAugment parameters for PyTorch and remove unnecessary functions.
- Match `eps` for LayerNorm with the Jax version.

### CODE REVIEW

File: algorithmic_efficiency/workloads/imagenet_resnet/imagenet_pytorch/randaugment.py [hunk 1]
```diff
@@ -168,6 +163,7 @@ def _augmentation_space(self) -> Dict[str, Tuple[Tensor, bool]]:
         "SolarizeAdd": (torch.tensor(110), False),
         "AutoContrast": (torch.tensor(0.0), False),
         "Equalize": (torch.tensor(0.0), False),
+        "Invert": (torch.tensor(0.0), False),
```
[the remark this answers is not in the document]
Comment (pomonam) [reply algorithmic_efficiency/workloads/imagenet_resnet/imagenet_pytorch/randaugment.py:166]: Thank you! I missed it haha, I will change it.

[the remark this answers is not in the document]
Comment (runame) [reply algorithmic_efficiency/workloads/imagenet_resnet/imagenet_pytorch/randaugment.py:166]: Haha the classic ;) Thanks!
